src/backend/aspen/app/views/tests/test_usergroup_views.py

```diff
@@ -12,7 +13,10 @@ def test_usergroup_view_get(session, app, client):
     with client.session_transaction() as sess:
         sess["profile"] = {"name": user.name, "user_id": user.auth0_user_id}
     res = client.get("/api/usergroup")
-    expected = {"user": user.to_dict(), "group": group.to_dict()}
+    expected = {
+        "user": filter_usergroup_dict(user.to_dict(), ["name", "agreed_to_tos"]),
+        "group": filter_usergroup_dict(user.group.to_dict(), ["name"]),
```

Comment:
These allowed fields should be constants (tuples?  frozensets?) in `src/backend/aspen/app/views/usergroup.py`.

Reply:
added some tuples!